### DISCUSSION TITLE
Logging unclear

### DISCUSSION THREAD

**Discussion by res1st** [Ulauncher/Ulauncher#1232] [Extensions and Themes]
Hi, started to develop an extension.

It is unclear for me, why I don't see any of my log messages.
I checked the file in ~/.local/share/ulauncher/last.log

I did that the following way
`logger = logging.getLogger(__name__)

...

def on_event(self, event, extension):
        logger.info('event UuidExt: %s', event)
`

It is also unclear how to enable a certain log level for my extension.
Maybe we could also improve the documentation in that point.

**Comment by friday** [comment]
Did you read https://docs.ulauncher.io/en/stable/extensions/debugging.html ?

**Comment by res1st** [comment]
Yes, there is not a single answer to my questions.
* I did it as it is documented, but there is not logging message for me
* Which log file do I have to check?
* Configuration of log levels?

**Comment by troycurtisjr** [comment]
I think the key is that it isn't in a log file, it is displayed to the terminal where ulauncher or extension is launched in dev mode, as shown in the debugging page.
